frontend/src/features/projects/spl/tabs/SplTab.tsx

```diff
@@ -5,6 +5,7 @@ import { ReviewWorkflowStatus } from 'features/projects/common';
 interface ISplTabProps {
   isReadOnly?: boolean;
   setSubmitStatusCode: Function;
+  goToGreTransferred: Function;
```

Comment:
is this different than declaring a prop as () => void ?

Reply:
Same thing - just a different way I believe